@kor1k | Coveralls add

kor1k opened the pull request:
added plugins in pom.xml coveralls and some in travis.yml

kor1k committed:
@kor1k | added pom.xml plugins for coveralls and some in travis

kor1k commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Add Coveralls](https://trello.com/c/c56hAikX/115-add-coveralls)

kor1k committed:
@kor1k | trying to make one simple test to make coveralls work

podolsky4 commented:
https://medium.com/@swhp/save-your-time-and-life-with-continuous-integration-bf755988ca13

kor1k committed:
@kor1k | added file .coveralls.yml, edited / added code to .travis.yml and pom.xml
__
deleted App.test.js (because it made conflicts)

kor1k committed:
Merge branch 'master' into korik-add-coveralls

kor1k commented:
Не работает из-за того, что это пулл #13 !!!

kor1k committed:
@kor1k | commented "after_success"

kor1k committed:
Merge remote-tracking branch 'origin/korik-add-coveralls' into korik-add-coveralls

podolsky4 committed:
Add simple unit test

podolsky4 committed:
Merge branch 'master' into korik-add-coveralls

podolsky4 committed:
Update .travis.yml

podolsky4 committed:
Update .travis.yml

podolsky4 committed:
Update coverall badge to default branch

podolsky4 committed:
Move test coverage report generation to `after_success`

Отчет о тестовом покрытии будет генерироваться в случае успешной сборки проекта

podolsky4 reviewed (approved):
Coverall is working

podolsky4 commented on .travis.yml:
```diff
@@ -1,25 +1,34 @@
-    - "$HOME/.m2 - ./frontend/node_modules/"
-
+    - $HOME/.m2 
+    - ./frontend/node_modules/
```
Каждый путь прописывайте в отдельной строке

podolsky4 commented on frontend/.coveralls.yml:
```diff
@@ -0,0 +1,3 @@
+#service_name: travis-pro
```
Этот файл нужно удалить, так  как он лежит в публичном репозитории

podolsky4 commented on pom.xml:
```diff
@@ -4,35 +4,20 @@
 
-    <groupId>ua.com.danit</groupId>
-    <artifactId>mango</artifactId>
+    <groupId>group-2</groupId>
```
Обычно maven groupId совпадает с названием пакета проекта, а у нас 'ua.com.danit'

podolsky4 commented on src/test/java/ua/com/danit/ApplicationTest.java:
```diff
@@ -0,0 +1,14 @@
+
+  @Test
+  public void testHelloWorld() {
+    System.out.println("Hello World");
```
Этот тест просто для галочки, реального тестового покрытия он не несет, дальше когда добавятся новые тесты, его придется удалить, чтобы не занимал места.

alexqwerty2000 committed:
Delete .coveralls.yml
